.azure-pipelines/ci-build.yml

```
@@ -58,4 +58,17 @@ jobs:
               publishVstsFeed: $(PROJECT_NAME)/$(PREVIEW_FEED_NAME)
               allowPackageConflicts: true
 
+          - task: PublishSymbols@2
+            displayName: Publish Symbols
+            inputs:
+              SymbolsFolder: '$(Build.ArtifactStagingDirectory)'
+              SearchPattern: '**//Microsoft.Graph.Entra.*.nupkg'
```

My understanding is that symbols packages are supposed to have a .snupkg suffix. Maybe that isn't a requirement. 

```suggestion
              SearchPattern: '**//Microsoft.Graph.Entra.*.snupkg'
```